src/site/facilities/health_service.drupal.liquid

```diff
@@ -24,7 +24,7 @@
                         {% for location in healthService.fieldLocalHealthCareService %}
                             {% assign facility = location.entity.fieldFacilityLocation.entity %}
                             {% if facility != empty %}<li class="vads-u-margin-bottom--2">
-                                <a onClick="recordEvent({ event: 'nav-accordion-embedded-link-click' });" href="{{ facility.entityUrl.path }}">{{ facility.fieldNicknameForThisFacility }}</a></li>{% endif %}
```

Comment:
This line looks fine to me. Why did we have to move into a build step?

Reply:
Because I need to target the wysiwyg output, which means parsing / manipulating a tags in it (can't add an onclick to an a tag in Drupal). This is where the output I need to target lives ```                {% if healthService.fieldBody.processed %}
                    <div>{{ healthService.fieldBody.processed }}</div>
                {% endif %}``` @ncksllvn

Reply:
Ah gotcha! Instead of a build step, can we instead do something like we for processing [popups](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/site-wide/popups.js), where we add event listeners from the browser?

Reply:
Good idea, and easy to do, but I'm concerned about adding an onclick event on the fly tracking properly in GA........should I not be, @ncksllvn ?

Reply:
Okay. Updated, @ncksllvn :+1: